logs/info.log

```diff
@@ -0,0 +1,55 @@
+[Thu Nov 22 2018 17:54:23 GMT+0100 (WAT)] : undefined
```

Comment:
We shouldn't be having these files here on Github, should we?

Reply:
It shouldn't, we will add it to gitignore. It was an oversight on our part. we will remove it